www/i18n/en.json

```diff
@@ -245,17 +245,28 @@
         }
     },
 
+    "intro": {
+        "permissions": {
+            "locationPermExplanation-android-lt-6": "you accepted the permission during installation. You don't need to do anything now.",
+            "locationPermExplanation-android-gte-6": "please select 'allow'",
+            "locationPermExplanation-ios-lt-13": "please select 'Always allow'. This allows us to protect you even when you are not actively using the app",
```

Comment:
Didn't you want to change "protect" here?

Reply:
'protect' is actually not a bad idea for now.. :)